install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -51,13 +50,36 @@ $ mount
 If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
 default value `0755` to `0777` or `0775`. This manual step is required because
 the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
-in {product-title} 3.9. The following examples show configuration files with the
-changed values.
+in {product-title} {product-version}. The following examples show configuration
```

Comment:
Revert this change. The version listed is relevant to the statement.

Reply:
Oh, oops. Thanks!